settings/EventCalendar.setting.php

```diff
@@ -102,6 +102,20 @@
     'help_text' => '',
     'html_attributes' => array('class' => 'four'),
   ),
+  'eventcalendar_event__type_filter' => array(
```

Comment:
eventcalendar_event__type_filter should be eventcalendar_event_type_filter remove extra _